src/constraint_system/composer.rs

```diff
@@ -322,7 +322,7 @@ impl TurboComposer {
         self.q_logic.push(BlsScalar::zero());
         self.q_fixed_group_add.push(BlsScalar::zero());
         self.q_variable_group_add.push(BlsScalar::zero());
-        self.q_lookup.push(BlsScalar::one());
+        self.q_lookup.push(BlsScalar::zero());
```

Comment:
That change makes sense to me. Was going to report in in the review `add_lookups` branch